src/components/NavigationPage/Navigation.tsx

```diff
@@ -0,0 +1,38 @@
+import cn from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation: React.FC = () => {
+  return (
```

Comment:
If your component returning JSX straight away, you don't need to write `return`, just use `(` instead of `{`
```suggestion
export const Navigation: React.FC = () => (
```